capstone/src/main/webapp/js/maps.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Displays the map onto the dashboard with hardcoded values.
+ * @return {void}
+ */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 34.052, lng: -118.245}, zoom: 10});
```

Comment:
Not important now but later when you are fetching data you could possibly center the map on the average latitude-longitude pair of the user's selected colleges.